gregberge/svgr
# Fix merging svgo config

## Summary



The only optimization we need in our project is prefixing ids. But when we tried to use the configuration below, we faced the problem - prefixes were added unexpectedly to the class names.

Use https://react-svgr.com/playground for reproducing the issue:

 - use simple svg example:
```
<svg height="100" width="100">
  <circle id="test" class="test" cx="50" cy="50" r="40" fill="red"/>
</svg>
```
 - add svgo config:
```
{ 
"full": true,
"plugins": [{ "prefixIds": {"prefixIds": true, "prefixClassNames": false} }]
}
```

 - check the output and note that both id and class have prefixes

This PR fixes the issue and allows to configure prefixIds plugin in more flexible way. 

## Test plan



New test case was added for plugin-svgo
